### ISSUE TITLE
Can I use this repo for unity script CI？

### ISSUE THREAD

**Issue by ffbh123456** [neogeek/unity-ci-tools#7]

**Comment by ffbh123456** [comment]
I use macos , can this repo work?

**Comment by ffbh123456** [comment]
I want run unity nunit test

**Comment by neogeek** [comment]
Absolutely! This repo should work with all known CI services. If you are looking to use it locally, it might make sense to use just a portion of the `test.sh` script. To make this process easier, I have released another project used for finding the application path of Unity via the command line for macOS.

https://github.com/neogeek/find-unity

Both that and the command found in `test.sh` will end up looking like this.

```bash
"$(find-unity)/Contents/MacOS/Unity" \
    -batchmode \
    -nographics \
    -silent-crashes \
    -stackTraceLogType Full \
    -logFile - \
    -projectPath "$(pwd)/" \
    -runEditorTests \
    -editorTestsResultFile "$(pwd)/test.xml"
```

**Comment by ffbh123456** [comment]
You are excellent ! it is work on my macOs, thank you for your help

**Comment by ffbh123456** [comment]
And it seems only support macOs ，Are you interested in being a linux version
